docker-compose.yml

```diff
@@ -45,7 +47,7 @@ services:
 
   registry:
     restart: always
-    image: registry:2.8.1
```

Comment:
Is it worth ensuring this is consistent with our other environments for debugging purposes?